src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Application {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the start of the class definition.